classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    super()
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    students << student
+    student.classroom = self
+  end
```

Comment:
- [ ] Good job creating this method, but we could improve our code by adding the @ before students to become @students << student when pushing into the @students array. 👍🏽

```ruby
  def add_student(student)
    @students << student
    student.classroom = self
  end
```